tests/test_schema.py

```diff
@@ -6,160 +6,62 @@
-            "warn_after": {
-                "count": "24",
-                "period": "hour"
-            },
-            "error_after": {
-                "count": "36",
-                "period": "hour"
-            }
-        },
-        "loaded_at_field": "last_login"
-    }
-    current_pii_downstream_source = {'name': 'THAT_TABLE', 'description': 'Expect this'}
-
-    schema.add_table_to_downstream_sources(
-        relation, current_safe_downstream_source, current_pii_downstream_source
-    )
-
-    expected_downstream_sources = {
-        "version": 2,
-        "sources": [
-            {
-                "name": 'LMS',
-                "database": 'PROD',
-                "tables": [
-                    {'name': 'THIS_TABLE', 'description': 'TODO: Replace me'},
-                    {
-                        'name': 'THAT_TABLE',
-                        'description': 'Make sure all of the aspects of this are preserved',
-                        "freshness": {
-                            "warn_after": {
-                                "count": "24",
-                                "period": "hour"
-                            },
-                            "error_after": {
-                                "count": "36",
-                                "period": "hour"
-                            }
-                        },
-                        "loaded_at_field": "last_login"
-                    }
-                ]
-            },
-            {
-                "name": 'LMS_PII',
-                "database": 'PROD',
-                "tables": [
-                    {'name': 'THIS_TABLE', 'description': 'TODO: Replace me'},
-                    {'name': 'THAT_TABLE', 'description': 'Expect this'}
-                ],
-            },
-        ],
-        "models": [],
-    }
-
-    assert schema.new_downstream_sources == expected_downstream_sources
+def test_raw_schema_filter_with_exclusion_list():
+    relations = [
+        Relation(
+            'THIS_TABLE', ['COLUMN_1', 'COLUMN_2'], 'LMS', 'models/PROD/LMS',
+            ['START', 'END'], [], []
+        ),
+        Relation(
+            'NOT_THIS_TABLE', ['COLUMN_1', 'COLUMN_2'], 'LMS', 'models/PROD/LMS',
+            ['START', 'END'], [], []
+        ),
+        Relation(
+            'THIS_TABLE_ALSO', ['COLUMN_1', 'COLUMN_2'], 'LMS', 'models/PROD/LMS',
+            ['START', 'END'], [], []
+        ),
+    ]
+    exclusion_list = ['NOT_THIS_TABLE']
+    raw_schema = Schema('LMS_RAW', exclusion_list, [], relations=relations)
+    filtered_relations = raw_schema.filter_relations()
+    expected_relations = [
+        Relation(
+            'THIS_TABLE', ['COLUMN_1', 'COLUMN_2'], 'LMS', 'models/PROD/LMS',
+            ['START', 'END'], [], []
+        ),
+        Relation(
+            'THIS_TABLE_ALSO', ['COLUMN_1', 'COLUMN_2'], 'LMS', 'models/PROD/LMS',
+            ['START', 'END'], [], []
+        ),
+    ]
+    # This is NOT a final draft
```

Comment:
Yeah looks like a little more work to be done here. I'd guess a check on the length being 2 would make it explicit what we're hoping for here.